src/generators/server-side-rendering/visitors/Element.js

```diff
@@ -11,6 +11,10 @@ export default {
 		let openingTag = `<${node.name}`;
 
 		node.attributes.forEach( attribute => {
+			if ( attribute.type === 'Spread' ) {
+				openingTag += ` \${Object.keys( root.${attribute.name} ).map( prop => \`\${prop}="\${root.${attribute.name}[prop]}"\` ).join( ' ' )}`;
```

Comment:
So this code was written a little later than the generator and while it works, I think it's really sloppy.